### PR TITLE
fix(es/AnimeFLV): change servers info

### PR DESCRIPTION
**Pull request by ludg1e**
Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [ ] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `containsNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

Changes:
- Removed DoodStream as it isn't available anymore as a server option
- Added StreamWish to the SERVER_LIST array (why it wasn't there?)
- Ordered as like on the website the SERVER_LIST array (optional)
- Changed PREF_QUALITY_DEFAULT to 480, as no server provides 1080p, and the only providing 480p and works well is StreamWish (Okru I think too)
- Changed PREF_SERVER_DEFAULT to StreamWish, as it's usually the best that works without that many ads, after MEGA
- Updated extension version to 53

---

- Removed maru server (doesn't even have a backend)
- Added a blank space to the SW extractor output

Non-related:
- Update Stape wording from StreamTape to Streamtape (lib)

### CODE REVIEW

**Comment by ludg1e** [comment]
Update: removed maru server (doesn't even have a backend) and added a blank space to the SW extractor output

TODO: add blank space to Okru extractor output

**Comment by ludg1e** [comment]
Upd to latest changes. Ready for review. It works (tested).

File: src/es/animeflv/src/eu/kanade/tachiyomi/animeextension/es/animeflv/AnimeFlv.kt [hunk 1]
```diff
@@ -50,12 +49,12 @@ class AnimeFlv : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
 
     companion object {
         private const val PREF_QUALITY_KEY = "preferred_quality"
-        private const val PREF_QUALITY_DEFAULT = "1080"
+        private const val PREF_QUALITY_DEFAULT = "480"
```
Comment (jmir1) [inline src/es/animeflv/src/eu/kanade/tachiyomi/animeextension/es/animeflv/AnimeFlv.kt:52]: Why did you change this line?

**Comment by ludg1e** [comment]
Ready for review and it works, I believe.

File: src/es/animeflv/src/eu/kanade/tachiyomi/animeextension/es/animeflv/AnimeFlv.kt (hunk 1 above)
Comment (jmir1) [reply src/es/animeflv/src/eu/kanade/tachiyomi/animeextension/es/animeflv/AnimeFlv.kt:52]: Nice, I can't read

**Review by jmir1 — APPROVED** [review]
